app/templates/info.html

```diff
@@ -1,5 +1,12 @@
 <main class="col-md-9 ms-sm-auto col-lg-10 px-md-4">
-  <a class="btn btn-secondary mt-1 add_button" role="button">Add</a>
+  {% if current_user.role.name == "wp_manager" %}
+  <a
+    class="btn btn-secondary mt-1 add_button"
+    href="{{ url_for('plan.work_add', ppc_type=ppc_type , type=request.args['type']) }}"
```

Comment:
```
href="{{ url_for('plan.work_add', ppc_type=ppc_type , type=request.args.get('type')) }}"
```
if not found, than ```None```

Reply:
it's ok, in logic after we handle this case